schema/user.schema.graphql

```diff
@@ -70,7 +70,7 @@ type User @entity {
   """
   EVM Bridge transfers
   """
-  crossChainTransfer: [CrossTransfer!] @derivedFrom(field: "rskUser")
+  crossChainTransfer: [CrossTransfer!] @derivedFrom(field: "user")
```

Comment:
didn't we talk about user and externalUser?

Reply:
I'll add external user, thanks

Reply:
@omerzam externalUser is added